### ISSUE TITLE
Image is only downloading at very first time but not next times

### ISSUE THREAD

**Issue by amitd123** [agap/picassos3#1]
after adding your library to my project i use below code snippet

  new Picasso.Builder(MainActivity.this).downloader(new S3Downloader(MainActivity.this, s3Client, bucket))
                .build()
                .load("https://s3-eu-east-5.amazonaws.com/music/Music_3.jpg")
                .noFade()
                .placeholder(R.drawable.img_placeholder)
                .memoryPolicy(MemoryPolicy.NO_CACHE)
                .networkPolicy(NetworkPolicy.NO_CACHE)
                .into(imageView);

with above code snippet image is loading in imageview but it works only first time after installing app and after that it's not loading the url image.

**Comment by sunwicked** [comment]
The next time the cache is not able to load the file from memory for some reason
```
if (cache.contains(uri)) {
          final InputStream in = cache.get(uri);
            if (in != null) {
                return new Response(in, true, cache.getSize(uri));
            }
}
```

return new Response is triggered but image is not loaded

**Comment by agap** [comment]
Closed via #3
